src/components/Cabinet/Cabinet.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Use lowercased name for the folder and file

Reply:
Renamed